code/modules/paperwork/contract.dm

```diff
@@ -160,6 +164,13 @@
 	else
 		info += "<i>[signature]</i>"
 
+/obj/item/paper/contract/infernal/pleasure/update_text(signature = "____________", blood = 0)
+	info = "<center><B>Contract for a friend</B></center><BR><BR><BR>I, [target] of sound mind, do hereby willingly offer my soul to the infernal hells by way of the infernal agent [devil_datum.truename], in exchange for boundless pleasure.  I understand that upon my demise, my soul shall fall into the infernal hells, and my body may not be resurrected, cloned, or otherwise brought back to life.  I also understand that this will prevent my brain from being used in an MMI.<BR><BR><BR>Signed, "
```

Comment:
zapomniałeś zmienić "contract for a friend"

Reply:
ups!

Reply:
naprawione